### PR TITLE
Remove DvdLib

### PR DESCRIPTION
**Pull request by barronpm**
Based on the investigation done in #5373, the current code for DvdLib is unused, and it doesn't work with Windows paths.

This PR removes the unused/broken code. 

@ferferga, the original PR also removed BDInfo, was that unused as well?

Replaces #5373

### CODE REVIEW

**Review by Shadowghost — APPROVED** [review]

**Comment by Shadowghost** [comment]
Afaik we're using ffmpeg for everything now.

**Comment by Roardom** [comment]
> Afaik we're using ffmpeg for everything now.

Is this why playback of DVDs has been broken since 10.7.0? (https://github.com/jellyfin/jellyfin/issues/7786)

**Review by Bond-009 — APPROVED** [review]

File: MediaBrowser.Providers/MediaInfo/FFProbeVideoInfo.cs [hunk 1]
```diff
@@ -85,40 +84,22 @@ public class FFProbeVideoInfo
...
                     var inputPath = item.Path;
-
                     blurayDiscInfo = GetBDInfo(inputPath);
 
-                    streamFileNames = blurayDiscInfo.Files;
-
-                    if (streamFileNames.Length == 0)
+                    if (blurayDiscInfo.Files.Length == 0)
                     {
                         _logger.LogError("No playable vobs found in bluray structure, skipping ffprobe.");
```
Comment (Shadowghost) [inline MediaBrowser.Providers/MediaInfo/FFProbeVideoInfo.cs:98]:
```suggestion
                        _logger.LogError("No playable filesfound in bluray structure, skipping ffprobe.");
```

**Review by Shadowghost — CHANGES_REQUESTED** [review]
After looking at the playback code and fixing metadata retrival for DVD folders locally I came to the conclusion that playing folders outside of an ISO was never supported for DVDs.

It works for BDs because FFmpeg provides a special protocol handler (`bluray://`) for those folders - which is not the case for DVDs. We'd need to handle VOB selection and concatenation ourself to make playing DVDs from folders work (which we currently do not at all).

While doing so I found another function you can remove (also needs to be removed from the interface):
`MediaBrowser.Encoding.Encoder.MediaEncoder.GetPrimaryPlaylistVobFiles`

**Review by crobibero — APPROVED** [review]

**Pull request merged**